mantle/kola/tests/coretest/core.go

```diff
@@ -378,3 +382,79 @@ func TestMachineID() error {
 	}
 	return nil
 }
+
+// TestRHCOSGrowpart tests whether rhcos-growpart.service was run successfully
+// and check that filesystem size has been grown to at least 15 GB.
+func TestRHCOSGrowpart() error {
+	// check that rhcos-growpart.service was run and exited normally
+	err := checkGrowpart("rhcos-growpart.service")
+	if err != nil {
+		return err
+	}
+
+	// check that filesystem size is >= 15 GB
+	// 16106127360 Bytes = 15 Gigabytes
+	err = checkFilesystemSize(16106127360)
```

Comment:
Minor: instead of the comment, we could rewrite as e.g. `15 * 1024 * 1024 * 1024`.